packages/dashboard/src/components/DashboardGridLayout/DashboardGridLayoutWithAddCardPanel/DraggingCard/DraggingCard.styles.ts

```diff
@@ -0,0 +1,47 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+
+export interface IAddCardStyles {
+  /**
+   * Style set for the add card item
+   */
+  container: IStyle;
+  title: IStyle;
+  titleText: IStyle;
+  body: IStyle;
+}
+
+export const getStyles = (): IAddCardStyles => {
+  return {
+    container: {
+      zIndex: 12000000,
+      touchAction: 'none',
+      position: 'absolute',
+      backgroundColor: 'rgb(255, 255, 255)',
+      borderRadius: '2px',
+      boxShadow: '0 2.4px 7.2px rgba(0, 0, 0, 0.18), 0 12.8px 28.8px rgba(0, 0, 0, 0.22)',
+      pointerEvents: 'none'
+    },
+    title: {
+      width: '100%-4px',
+      height: '20px',
+      borderTopLeftRadius: '2px',
+      borderTopRightRadius: '2px',
+      borderBottom: '1px solid rgb(0, 0, 0, 0.1)',
+      position: 'relative',
+      color: 'rgb(0, 0, 0)',
+      fontFamily: 'Segoe UI',
+      fontWeight: '600',
+      lineHeight: '16px',
+      fontSize: '14px !important',
```

Comment:
why important for font size and also use fabric

Reply:
Changed to fabric font